cwxml/ytyp.py

```diff
@@ -59,7 +58,7 @@ def __init__(self):
         self.extensions = ExtensionsList()
         if current_game() == SollumzGame.RDR:
             self.guid = ValueProperty("guid")
-            self.unknown_1 = TextProperty("iypiqkia_0x07d164a8")
+            self.unknown_1 = TextProperty("zqNiUDA_0x07D164A8")
```

Comment:
The string we had here was the one generated when CX has the lowercase hash collisions option enabled. Ideally we should support both (though we don't have a good way to do this currently), or solve the hash I guess.